libr/anal/p/anal_mips_cs.c

```diff
@@ -82,11 +82,20 @@ static ut64 t9_pre = UT64_MAX;
 #define ES_J(addr) addr",pc,="
 #endif
 
+#define ES_B(x) "0xff,"x",&"
+#define ES_H(x) "0xffff,"x",&"
+#define ES_W(x) "0xffffffff,"x",&"
+
 // sign extend 32 -> 64
-#define ES_SIGN_EXT64(arg) \
-	arg",0x80000000,&,0,<,?{,"\
-		"0xffffffff00000000,"arg",|=,"\
-	"}"
+
+#define ES_SIGN_n_64(arg, n_bit)  do{if (a->bits == 64) \
```

Comment:
Make a separate function or properly indent this as multiline

Reply:
I want to save a little function call expend

Reply:
@junchao-loongson you can just add an inline function instead.

Reply:
okay~